src/rand.rs

```diff
@@ -270,6 +270,17 @@ mod sysrand_chunk {
     }
 }
 
+#[cfg(all(target_arch = "wasm32", target_os = "wasi",))]
+mod sysrand_chunk {
+    use crate::error;
+
+    #[inline]
+    pub fn chunk(dest: &mut [u8]) -> Result<usize, error::Unspecified> {
+        wasi::wasi_unstable::random_get(dest).map_err(|_| error::Unspecified)?;
```

Comment:
It seems like, rather than adding that heavyweight dependency, we should just call `__wasi_random_get` directly and avoid that crate.